packages/Webkul/Shop/src/Resources/views/checkout/onepage.blade.php

```diff
@@ -445,21 +445,18 @@
                 return {
                     templateRender: null,
 
-                    selected_shipping_method: '',
-
-                    first_iteration : true,
+                    selected_shipping_method: ''
                 }
             },
 
             staticRenderFns: shippingTemplateRenderFns,
 
             mounted: function() {
                 for (method in shippingMethods) {
-                    if (this.first_iteration) {
-                        for (rate in shippingMethods[method]['rates']) {
+                    for (rate in shippingMethods[method]['rates']) {
+                        if (shippingMethods[method].default) {                            
```

Comment:
The problem actually lies here, this is getting data from DB, not from the config which returning undefined.